### ISSUE TITLE
Mouse Cursor Constantly On Screen

### ISSUE THREAD

**Issue by onangelswings** [willis81808/Mache#4]
After the latest update to sons of the forest, while using Mache, a secondary mouse cursor is persistently on the screen with no way of disabling it.

nothing in the mache menu to disable this either and is causing issues while playing.

**Comment by realrobd** [comment]
Same here.
Removing **UniverseLib.IL2CPP.Interop** fixes the issue.

**Comment by onangelswings** [comment]
I do not believe i have that downloaded though, as its a extra modding library isnt it ?

**Comment by realrobd** [comment]
> 

It comes with the Mache framework download.

**Comment by MacedonskyAlex** [comment]
+

**Comment by onangelswings** [comment]
Removing the UniverseLib.IL2CPP.Interop fixes the mouse cursor, however cant open the mache config now for the warps mod

**Comment by JoewAlabel** [comment]
Same here.
Some time ago i can hide the mouse cursor open menu and after pressing ESC key. But now it does not hide anymore.
Any way to fix it, please?
Ty!!!!!

**Comment by EoLykos** [comment]
> Same here. Removing **UniverseLib.IL2CPP.Interop** fixes the issue.

I'm guessing this was the file in the plugins folder? UniverseLibIL2CPP.Interop.dll, I tried removing this and I still get the extra mouse cursor that won't go away :(

**Comment by EoLykos** [comment]
Anyone find a work around for this ?
